app/controllers/admin/positions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::PositionsController < ApplicationController
+  before_action -> { user_has_position(Position.first) }
```

Comment:
According to the seed file, the secretary is always first in the db. and when the secretary is updated, only the user_id will be changed leaving it still as the first object in the position table.